docs/_posts/2020-05-25-jekyll-4-1-0-released.markdown

```diff
@@ -45,6 +45,14 @@ released gem, please remove `|_config\.yml` from the regular expression in the g
 [filters-docs]: {{ 'docs/liquid/filters/' | relative_url }}
 [page-drop-docs]: {{ 'docs/pages/#for-plugin-developers' | relative_url }}
 
+
+### Have questions?
+
+Please reach out on our [community forum](https://talk.jekyllrb.com)
```

Comment:
👍

Reply:
Found this in the Releast Post for `v4.0.0` :wink: